Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/IO/polygon_mesh_io.h

```diff
@@ -37,7 +37,8 @@ namespace IO {
 /*!
   \ingroup PMP_IO_grp
 
- * \brief reads the file as a polygon soup, repairs, and orients it as to obtain a polygon mesh.
+ * \brief reads the file as a polygon soup, repairs (using `repair_polygon_soup()`),
+ * and orients it (using `orient_polygon_soup()`) as to obtain a polygon mesh.
```

Comment:
I would still suggest to add a sentence like     "If there is no need to orient and repair the input you should use the function   `CGAL::IO::read_polygon_mesh()`"